test/unit/helpers-test.js

```diff
@@ -16,16 +16,5 @@ describe('helpers', function() {
         assert.equal(tester[0], 'hello');
         assert.equal(tester[1], 'hi');
     });
-
-    it('log will not thow if console is cleared', function() {
-        var tmpConsole = window.console;
-
-        window.console = null;
-        // this should not break
-        utils.log('testing');
-
-        // restore actual window console
-        window.console = tmpConsole;
-    });
```

Comment:
Synched up with @johnBartos about this and looks like there's no straightforward way to test this. Don't want to sink in anymore time right now.